scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -36,28 +36,208 @@ import { Root } from "mdast";
 export async function sphinxHtmlToMarkdown(options: {
   html: string;
   url: string;
-  imageDestination?: string;
+  imageDestination: string;
   // url links to a fixed version and ending in /
   // https://github.com/Qiskit/qiskit-ibm-runtime/tree/0.9.2/
-  baseSourceUrl?: string;
-  releaseNotesTitle?: string;
+  baseSourceUrl: string;
+  releaseNotesTitle: string;
```

Comment:
These were only optional for the sake of testing. But that's a dangerous strategy because it's really important that production never forget to set the arguments.